<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>subWindow</class>
 <widget class="QDialog" name="subWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>675</width>
    <height>335</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>10</y>
     <width>131</width>
     <height>271</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_5">
    <item alignment="Qt::AlignHCenter">
     <widget class="QLabel" name="label_7">
      <property name="font">
       <font>
        <pointsize>11</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Transaction 2</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="label_8">
      <property name="font">
       <font>
        <pointsize>9</pointsize>
       </font>
      </property>
      <property name="text">
       <string>No.of data phases</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QSpinBox" name="Trans2_Spinbox"/>
    </item>
    <item>
     <widget class="QComboBox" name="Trans2_comboBox"/>
    </item>
    <item>
     <widget class="QLabel" name="label_9">
      <property name="text">
       <string>Byte Enable</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QCheckBox" name="Trans2_Byte1">
      <property name="text">
       <string>Byte 1</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QCheckBox" name="Trans2_Byte2">
      <property name="text">
       <string>Byte 2</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QCheckBox" name="Trans2_Byte3">
      <property name="text">
       <string>Byte 3</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QCheckBox" name="Trans2_Byte4">
      <property name="text">
       <string>Byte 4</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="layoutWidget_2">
   <property name="geometry">
    <rect>
     <x>330</x>
     <y>10</y>
     <width>131</width>
     <height>271</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_6">
    <item alignment="Qt::AlignHCenter">
     <widget class="QLabel" name="label_10">
      <property name="font">
       <font>
        <pointsize>11</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Transaction 3</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="label_11">
      <property name="font">
       <font>
        <pointsize>9</pointsize>
       </font>
      </property>
      <property name="text">
       <string>No.of data phases</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QSpinBox" name="Trans3_Spinbox"/>
    </item>
    <item>
     <widget class="QComboBox" name="Trans3_comboBox">
      <item>
       <property name="text">
        <string>Data Phase 1</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Data Phase 2</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Data Phase 3</string>
       </property>
      </item>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="label_12">
      <property name="text">
       <string>Byte Enable</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QCheckBox" name="Trans3_Byte1">
      <property name="text">
       <string>Byte 1</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QCheckBox" name="Trans3_Byte2">
      <property name="text">
       <string>Byte 2</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QCheckBox" name="Trans3_Byte4">
      <property name="text">
       <string>Byte 3</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QCheckBox" name="Trans3_Byte4_2">
      <property name="text">
       <string>Byte 4</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="layoutWidget_3">
   <property name="geometry">
    <rect>
     <x>480</x>
     <y>10</y>
     <width>131</width>
     <height>271</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_7">
    <item alignment="Qt::AlignHCenter">
     <widget class="QLabel" name="label_13">
      <property name="font">
       <font>
        <pointsize>11</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Transaction 4</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="label_14">
      <property name="font">
       <font>
        <pointsize>9</pointsize>
       </font>
      </property>
      <property name="text">
       <string>No.of data phases</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QSpinBox" name="Trans4_Spinbox"/>
    </item>
    <item>
     <widget class="QComboBox" name="Trans4_comboBox">
      <item>
       <property name="text">
        <string>Data Phase 1</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Data Phase 2</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Data Phase 3</string>
       </property>
      </item>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="label_15">
      <property name="text">
       <string>Byte Enable</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QCheckBox" name="Trans4_Byte1">
      <property name="text">
       <string>Byte 1</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QCheckBox" name="Trans4_Byte2">
      <property name="text">
       <string>Byte 2</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QCheckBox" name="Trans4_Byte3">
      <property name="text">
       <string>Byte 3</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QCheckBox" name="Trans4_Byte4">
      <property name="text">
       <string>Byte 4</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>10</y>
     <width>131</width>
     <height>271</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_4">
    <item alignment="Qt::AlignHCenter">
     <widget class="QLabel" name="label_6">
      <property name="font">
       <font>
        <pointsize>11</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Transaction 1</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="label_4">
      <property name="font">
       <font>
        <pointsize>9</pointsize>
       </font>
      </property>
      <property name="text">
       <string>No.of data phases</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QSpinBox" name="Trans1_Spinbox"/>
    </item>
    <item>
     <widget class="QComboBox" name="Trans1_comboBox"/>
    </item>
    <item>
     <widget class="QLabel" name="label_5">
      <property name="text">
       <string>Byte Enable</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QCheckBox" name="Trans1_Byte1">
      <property name="text">
       <string>Byte 1</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QCheckBox" name="Trans1_Byte2">
      <property name="text">
       <string>Byte 2</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QCheckBox" name="Trans1_Byte3">
      <property name="text">
       <string>Byte 3</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QCheckBox" name="Trans1_Byte4">
      <property name="text">
       <string>Byte 4</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
